normandy/capabilities/api/v3/views.py

```diff
@@ -17,7 +17,7 @@ def get(self, request):
             capabilities[cap] = {"is_baseline": True}
 
         for recipe in Recipe.objects.all():
-            for cap in recipe.capabilities:
+            for cap in recipe.latest_revision.capabilities:
```

Comment:
I wasn't 100% sure about the implications for this API.

Reply:
This API isn't really used much, so I think it is fine. I think the more correct thing would be to iterate over all revisions, but I don't expect capabilities to vary much over time so it shouldn't make any difference.

That being said, this whole API isn't that useful, because what we should be doing is pulling this from the build artifacts generated by Firefox now. That's future work though. I think that this is good as it is.